astropy/modeling/tests/test_functional_models.py

```diff
@@ -235,8 +235,8 @@ def test_Ellipse2D():
     rotation = models.Rotation2D(angle=theta.degree)
     point1 = [2, 0]  # Rotation2D center is (0, 0)
     point2 = rotation(*point1)
-    point1 = np.array(point1) + [x0, y0]
-    point2 = np.array(point2) + [x0, y0]
+    point1 = [*np.array(point1), x0, y0]
+    point2 = [*np.array(point2), x0, y0]
```

Comment:
This is completely wrong. This was never a concatenation, but a sum of two arrays.

Reply:
Pushing a fix.